services/director-v2/src/simcore_service_director_v2/api/routes/services.py

```diff
@@ -29,42 +26,42 @@ async def list_services(
             "   - interactive - an interactive service\n"
         ),
     ),
-    forward_request: Coroutine = Depends(get_request_to_director_v0),
+    forward_request: Response = Depends(forward_to_director_v0),
 ):
-    return await forward_request()
+    return forward_request
```

Comment:
I still do not like this ... but i accept it. At least rename as ``response_from_forward`` or something like that

Reply:
this will not stay here. once the old director goes into retirement this will disappear